sigstore/_internal/oidc/ambient.py

```diff
@@ -27,10 +27,10 @@
 
 logger = logging.getLogger(__name__)
 
-GCP_PRODUCT_NAME_FILE = "/sys/class/dmi/id/product_name"
-GCP_TOKEN_REQUEST_URL = "http://metadata/computeMetadata/v1/instance/service-accounts/default/token"  # noqa # nosec B105
-GCP_IDENTITY_REQUEST_URL = "http://metadata/computeMetadata/v1/instance/service-accounts/default/identity"  # noqa # nosec B105
-GCP_GENERATEIDTOKEN_REQUEST_URL = "https://iamcredentials.googleapis.com/v1/projects/-/serviceAccounts/{}:generateIdToken"  # noqa
+_GCP_PRODUCT_NAME_FILE = "/sys/class/dmi/id/product_name"
+_GCP_TOKEN_REQUEST_URL = "http://metadata/computeMetadata/v1/instance/service-accounts/default/token"  # noqa # nosec B105
+_GCP_IDENTITY_REQUEST_URL = "http://metadata/computeMetadata/v1/instance/service-accounts/default/identity"  # noqa # nosec B105
+_GCP_GENERATEIDTOKEN_REQUEST_URL = "https://iamcredentials.googleapis.com/v1/projects/-/serviceAccounts/{}:generateIdToken"  # noqa
```

Comment:
Tests that monkeypatch these need updated as well.

Reply:
Whoops, thanks!